sdk/communication/communication-identity/test/communicationIdentityClient.mocked.spec.ts

```diff
@@ -72,12 +72,12 @@ describe("CommunicationIdentityClient [Mocked]", () => {
 
   it("[createUser] excludes _response from results when stringified", async () => {
     const client = new TestCommunicationIdentityClient();
-    const user = await client.createUserTest();
+    const customUser = await client.createUserTest();
```

Comment:
Why the rename?

Reply:
The linter was complaining thinking `user` was a shadowed variable (it is defined up top) so I just did a rename to silence the warning